layouts/millennium/r4/bulk-data.html

```diff
@@ -0,0 +1,7 @@
+<p>The FHIR Bulk Data Access specification provides system applications the ability to access large volumes of core clinical data for a group of individuals. Common use cases include exporting data to populate a research database, a health registry or exporting clinical data to calculate quality measures for a patient population.</p>
```

Comment:
We should probably use the current market brand name in the 1st sentence: _Cerner Ignite APIs_ for Millennium with Bulk Data Access. Also after the word individuals in the 1st sentence add "in alignment with standards cited in 21st Century Cures Act regulations."

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/899/commits/ee37ee8e4678c8be7160f7ce7d603ef63930b8ac#diff-4964db5e96ab4058c305df04ae967f586bc3105f3fad1b8ee39baafa3ea9626dR1)